### PR TITLE
Updating the axum integration doc (issue #588)

### PR DESCRIPTION
**Pull request by DGotshalk**
When using the askama feature "with-axum" you need to also depend on the "askama_axum" crate directly.

### CODE REVIEW

File: book/src/integrations.md [hunk 1]
```diff
@@ -22,7 +22,7 @@ from the Askama test suite for more on how to integrate.
 
 ## Axum integration
 
-Enabling the `with-axum` feature appends an implementation of Axum's
+Enabling the `with-axum` feature and depending on the `askama_axum` crate appends an implementation of Axum's
```
Comment (djc) [inline book/src/integrations.md:25]: If you depend on askama_axum I don't think you have specifically enable the `with-axum` flag, have you tried that?

Comment (DGotshalk) [reply book/src/integrations.md:25]: If the flag is not there, it doesn't seem to use the askama_axum crate at all. Whatever axum specific integration you have done with the askama_axum crate isn't seemingly touched without that flag

Comment (DGotshalk) [reply book/src/integrations.md:25]: Hello,
I just tried it. If I remove the `with-axum` flag, then having the `askama_axum` crate seemingly makes no difference?

Comment (DGotshalk) [reply book/src/integrations.md:25]: Yea, can confirm that. 

If you have the `with-axum`  feature, you need to have that dependency. However,  askama seems to works with axum even without the `askama_axum` crate? 

I am confused!

Comment (djc) [reply book/src/integrations.md:25]: I'm sorry, I don't have much time to dig into this. The feature flag is forward to the askama_derive crate, which in its `generator` generates some source code specific to the Axum integration. For many of the integrations, the generated source code depends on the integration crate but that may not be true here? Please have a look at the relevant source code yourself.

Comment (DGotshalk) [reply book/src/integrations.md:25]: Sure thing, I'll use that information to the best of my abilities.

Comment (DGotshalk) [reply book/src/integrations.md]: I can confirm that when the `with-axum` flag is used, the generated source code depends on the `askama_axum` integration crate. 

I can also confirm, that without the `with-axum` feature flag, the integration code is still generated. I can call the `into_reponse` function that is defined in the `askama_axum` on a Template.

**Comment by DGotshalk** [comment]
I can confirm that when the with-axum flag is used, the generated source code depends on the askama_axum integration crate.

I can also confirm, that without the with-axum feature flag, the integration code is still generated. I can call the into_reponse function that is defined in the askama_axum on a Template.

If you check integration doc files now, I have updated the pull request file to say that you only need to depend on the askama_axum crate, and don't need the feature flag. I don't know if you want the feature flag mentioned in the integration docs.

Also interesting note. I cannot depend on askama_axum only. When using the Templates, it still looks for the askama crate in my dependencies even though it should get re-exported as a part of askama_axum but that may be an issue for another pull request?

**Pull request closed without merging**